roles/docker/tasks/package_docker.yml

```diff
@@ -32,8 +32,11 @@
 # Make sure Docker is installed, but does not update a running version.
 # Docker upgrades are handled by a separate playbook.
 - name: Install Docker
-  package: name=docker{{ '-' + docker_version if docker_version is defined else '' }} state=present
   when: not openshift.common.is_atomic | bool
+  package: name=docker{{ '-' + docker_version if docker_version is defined else '' }} state=present
+  register: l_result
+  retries: 4
+  until: l_result.rc is defined and l_result.rc == 0
```

Comment:
I'm curious if `until` statements follow the same `AND`'ing pattern that `when` statements have. I.e., separable into a multi-line list